packages/suite-web-landing/utils/switchFavicon.ts

```diff
@@ -15,8 +15,11 @@ export const switchFavicon = () => {
             appleIcon?.setAttribute('href', resolveStaticPath('/images/icons/favicon/favicon.png'));
         }
     };
-    window
-        .matchMedia('(prefers-color-scheme: dark)')
-        .addEventListener('change', e => switchIcons(e.matches));
+    const matchMedia = window.matchMedia('(prefers-color-scheme: dark)');
```

Comment:
I think this could be turned into a hook, also check [this](https://usehooks.com/useDarkMode/) :)

Reply:
unrelated to this issue